packages/module/src/TagCount/TagCount.tsx

```diff
@@ -6,14 +6,18 @@ import { createUseStyles } from 'react-jss'
 
 const useStyles = createUseStyles({
   buttonTagCount: {
-    color: 'var(--pf-v5-global--icon--Color--light)',
     display: 'flex',
-    alignItems: 'center'
+    alignItems: 'center',
+    padding: 'var(--pf-t--global--spacer--200) var(--pf-t--global--spacer--300)'
   },
 
+  tagIcon: (isDisabled: boolean) => ({
+    color: `var(--pf-t--global--icon--color--${isDisabled ? '200' : '100'})`,
+  }),
+
   tagText: {
-    marginLeft: 'var(--pf-v5-global--spacer--sm)',
-    fontSize: 'var(--pf-v5-global--FontSize--sm)'
+    marginLeft: 'var(--pf-t--global--spacer--200)',
+    fontSize: 'var(--pf-t--global--font--size--200)'
```

Comment:
Wouldn't it better to use `var(--pf-t--global--spacer--md)` and `var(--pf-t--global--FontSize--sm)`?